docs/training/apis-howto/api-core-mnist-ug.rst

```diff
@@ -0,0 +1,573 @@
+#####################
+ Core API User Guide
+#####################
+
+This guide will help you get up and running with the Core API.
+
++------------------------------------------------------------------+
+| Visit the API reference                                          |
++==================================================================+
+| :doc:`/reference/reference-training/training/api-core-reference` |
++------------------------------------------------------------------+
+
+In this user guide, we'll show you how to adapt model training code to use the Core API. As an
+example, we'll be working with the PyTorch MNIST model.
+
+************
+ Objectives
+************
+
+The step-by-step instructions walk you through modifying a script for the purpose of performing the
+following functions:
+
+-  Report metrics
+-  Report checkpoints
+-  Perform a hyperparameter search
+-  Perform distributed training
+
+After completing the steps in this user guide, you will be able to do the following:
+
+-  Understand the minimum requirements for running an experiment
+-  Modify a script and an experiment configuration file
+-  Understand how to convert model code
+-  Use the Core API to train a model
+
+***************
+ Prerequisites
+***************
+
+**Required**
+
+-  A Determined cluster
+
+**Recommended**
+
+-  :doc:`Quickstart for Model Developers </quickstart-mdldev>`
+
+*****************************************************
+ Step 1: Get the Tutorial Files & Run the Experiment
+*****************************************************
+
+To run an experiment, you need, at minimum, a script and an experiment configuration (YAML) file.
```

Comment:
Do we need to say `(YAML)` all the time?

I feel like we've not been saying that for years now, and I don't recall anybody being confused by that.

Reply:
(I'll say, this sentence is perhaps the only place in this document where explicitly saying yaml seems ok)

Reply:
resolved, updated style guide